### ISSUE TITLE
NSP vulnerability for protobufjs <=6.8.5

### ISSUE THREAD

**Issue by mboughaba** [firebase/firebase-js-sdk#872]
### [REQUIRED] Describe your environment

  * Firebase SDK version: 5.0.4
  * Firebase Product: firestore

#### Steps to reproduce:

```shell
nsp check
```

I am referencing #813. It looks like this is still an issue. 
`protobufjs@5.0.3` seems vulnerable too.

```shell
λ nsp check --reporter table
(+) 1 vulnerability found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Denial of Service                                                  │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ protobufjs                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 5 (Medium)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 5.0.3                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <=6.8.5                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=6.8.6                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ alita-web@0.1.2 > firebase@5.0.4 > @firebase/firestore@0.5.4 >     │
│            │ grpc@1.11.3 > protobufjs@5.0.3                                     │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/605                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

Cheers

**Comment by omar10594** [comment]
The fix was ported to 5.x.x version, 5.0.3 has the fix.
But both nodesecurity still have outdated information on the status of this problem.
After update to 5.0.3 you can ignore the vulnerability until nodesecurity update their database.

see dcodeIO/protobuf.js/pull/1030

**Comment by mboughaba** [comment]
nsp database has been update and indeed fix was ported to 5.0.3.
